graylog2-web-interface/src/components/maps/configurations/GeoIpResolverConfig.tsx

```diff
@@ -17,19 +17,22 @@
 import React, { useEffect, useState } from 'react';
 import { Field, Form, Formik } from 'formik';
 
-import { IfPermitted, Select } from 'components/common';
+import { IfPermitted, Select, TimeUnitInput } from 'components/common';
 import { Button, Col, Input, Modal, Row } from 'components/bootstrap';
 import FormikInput from 'components/common/FormikInput';
 import { DocumentationLink } from 'components/support';
 
 export type GeoVendorType = 'MAXMIND' | 'IPINFO'
+export type TimeUnit = 'SECONDS' | 'MINUTES' | 'HOURS' | 'DAYS'
```

Comment:
@danotorrey Should we include less/more units--or for that matter, limit at all?

Reply:
This list sounds good to me.